src/lib/schema/feature-schema.ts

```diff
@@ -35,7 +36,25 @@ export const variantsSchema = joi.object().keys({
         .object()
         .keys({
             type: joi.string().required(),
-            value: joi.string().required(),
+            value: joi
```

Comment:
Should we move this Joi logic into its own `const variantValueSchema`? Just for readability.

Kind of like https://github.com/Unleash/unleash/blob/main/src/lib/routes/util.ts#L6-L20

Reply:
Sure, I've created a new constant in the same `feature-schema.ts` file where I moved just the variant value joi validation